### ISSUE TITLE
Netflix plugin stopped working

### ISSUE THREAD

**Issue by pszafer** [asciidisco/plugin.video.netflix#246]
*I'm submitting a ...*
  - [x] bug report

## General infomration
Yesterday I watched one episode and it was working.
Then second suddenly, stopped working.
Nothing updated

* [ ] Are you running the latest version?
inputstream.adaptive 2.1.4 on LibreElec.
just upgraded to version 2.1.8, same issue.

### Steps to Reproduce
Play any movie

### Installation
* [ ] I installed the plugin via zip from the Releases page
* [x] I´m using the Netflix Repo
* [ ] I´m using a different source (Please tell which)

### Operating System
* [ x ] Linux (ARM)

#### Additional informatin on the environment
LibreElec from Raybuntu for Odroid C2

## Debug log
http://sprunge.us/WIOP

**Comment by pszafer** [comment]
debug log: http://sprunge.us/HPLW

**Comment by pszafer** [comment]
Ok. 
Got it, I changed password to netflix account and I don't know how it works but what I know.

I have message from netflix:

~~~
15:50:53.838 T:3832542096   DEBUG: [Netflix] Error getting Manifest: {"signature":"ko9vhij9W5USYyPlN7Lz2R3mUPbM3x9bPpZvQ3Pia00T5HxEIIMZsxb+voCakhYd3BDoOWmmXS3jxi3UKyr1YQdWv1HL6dHlQdCbgxeRru3G/dO4Mekeb3sfYrbsJiZvL1pGKNoON6lNuG+bU8tJdcOz2QVKnx9QcI2+0EHvcwbfDe4gXNHTZ4ZDDb1bjEOzrvaFtSYYoXcM+GfBUUjptYHJnKcMiJij0jCJcBiHuprQXeSK0nt9kkhnF77Z6n1CYKKHFA06AjV9d9o8Jv/UaYMWAbThU0gXB6Fyu2e8E125nR9Rx4cRM6PFpye4gP++IgKq/2IpW1JcCc3BItJbLw==","errordata":"eyJpbnRlcm5hbGNvZGUiOjIwNTAwNywibWVzc2FnZWlkIjo0MDYyMjk1NDQzNDA1NDcwLCJlcnJvcmNvZGUiOjcsImVycm9ybXNnIjoiRW1haWwgb3IgcGFzc3dvcmQgaXMgaW5jb3JyZWN0LiIsInRpbWVzdGFtcCI6MTUxMTg4MDY1M30=","entityauthdata":{"scheme":"RSA","authdata":{"identity":"Netflix","pubkeyid":"MSL_TRUSTED_NETWORK_SERVER_KEY"}}}
~~~

After encoding error data:

~~~
{"internalcode":205007,"messageid":4062295443405470,"errorcode":7,"errormsg":"Email or password is incorrect.","timestamp":1511880653}
~~~

So I logged out then from netflix and logged with new credentials but it seems there were some files in 

~~~
~/.kodi/cdm/
~~~

.challenge, .init, .response which hold old credentials. I deleted those, and logged in again and it is working!

So I think plugin could at least decode error message and print it to us.
